gtdynamics.i

```diff
@@ -200,6 +197,11 @@ class Link  {
   gtsam::Matrix6 inertiaMatrix();
 
   void print(const std::string &s = "") const;
+
+  static gtdynamics::Link fix(const gtdynamics::Link& link);
```

Comment:
Why expose them in first place if they are private?

Reply:
This is the static (functional) version.